src/util/transaction/transactionUtils.ts

```diff
@@ -121,5 +124,5 @@ export function combineAndRegroupSignerTxns(
  * @returns the account address of the sender
  */
 export function extractSenderAddressFromTransaction(txn: Transaction): string {
-  return algosdk.encodeAddress(txn.from.publicKey);
+  return txn.sender.toString();
 }
```

Comment:
After the changes, this custom function doesn't provide any value over directly using `txn.sender.toString()`, we can remove this and replace usages simply with `txn.sender.toString()`